### PR TITLE
rock breaker behavior enhancement

### PR DESCRIPTION
**Pull request by POPlol333**
closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10812

### CODE REVIEW

File: src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java [hunk 1]
```diff
@@ -778,9 +778,10 @@ public void run() {
...
-        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.wireGt01, Materials.SuperconductorUEVBase, 27L), GT_OreDictUnificator.get(OrePrefixes.pipeTiny, Materials.Infinity, 18L), ItemList.Electric_Pump_UEV.get(1L), GT_Utility.getIntegratedCircuit(9)}, Materials.Helium.getGas(28000L), GT_OreDictUnificator.get(OrePrefixes.wireGt01, Materials.SuperconductorUEV, 27L), 3200, 7864320);
-        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.wireGt01, Materials.SuperconductorUMVBase, 33L), GT_OreDictUnificator.get(OrePrefixes.pipeTiny, Materials.SpaceTime, 22L), ItemList.Electric_Pump_UMV.get(1L), GT_Utility.getIntegratedCircuit(9)}, Materials.Helium.getGas(36000L), GT_OreDictUnificator.get(OrePrefixes.wireGt01, Materials.SuperconductorUMV, 33L), 3200, 125829120);
-
+        if(Loader.isModLoaded("GalaxySpace")) {
```
Comment (Glease) [inline src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java:781]: why this? seems totally unrelated.

**Review by Glease — CHANGES_REQUESTED** [review]

File: src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java (hunk 1 above)
Comment (POPlol333) [reply src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java:781]: just didn't head reset, this come from here: https://github.com/GTNewHorizons/GT5-Unofficial/pull/1231

Comment (POPlol333) [reply src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java:781]: this has already been merged btw
